src/selectors-api/selectors-api.json

```diff
@@ -0,0 +1,45 @@
+{
+    "badge": "badge",
+    "breadcrumbs": "breadcrumbs",
+    "button": "btn",
+    "fake-button": "fake-btn",
```

Comment:
If we are using arrays, we should probably group `fake-btn` under button. However, I think the current array format, based on an ordering convention, could prove to be a little brittle. We may either want an extra level of key value pairs, or just remove the arrays altogether and create a flat list. The first option will nicely mirror our module class hierarchy, whereas the second option feels a little simpler. Maybe worth asking the consuming team which they'd prefer.